v-next/hardhat-node-test-reporter/integration-tests/fixture-tests/execution-failure-test/test.ts

```diff
@@ -0,0 +1 @@
+throw new Error("error");
```

Comment:
Note for a future pr: we need to change the output generated in this case, as it has nothing to do with promises.